### DISCUSSION TITLE
Disable Native HTML5 date picker

### DISCUSSION THREAD

**Discussion by alexplex** [elementor/elementor#23125] [Developers Community]
Hi guys! So, I understand that Elementor forms uses the Flatpickr library for their date fields. And it seems that Flatpickr has enabled the native HTML5 date picker by default in mobile view via this option: 

`disableMobile: "false"`

Elementor support tells me that there is not way to change this to true without custom code. I would very much like to disable the native HTML5 date picker on mobile view since it doesn't support placeholders and any CSS workaround to that does not work for me. 

Since I'm not a programmer, I asked ChatGPT to write a custom code for me that would change that option to true with this code.

```
<script>
  document.addEventListener('DOMContentLoaded', function () {
    // Check if Elementor and Flatpickr are loaded
    if (window.elementor && window.flatpickr) {
      // Modify Flatpickr defaults to disable mobile
      flatpickr.defaultConfig.disableMobile = true;
    }
  });
</script>
```

But the Custom Code warns that 'flatpickr' is not defined.

Anyway, that is what I've tried but I have no idea what I'm doing. Does anyone have any idea? I would greatly appreciate it.

**Comment by hitsgee** [comment]
There is no default hook or function to override this in elementor as it's an external resource loaded. You could modify the flatpickr.js file or create a custom JS function to add this but it probably is not recommended. Why do you need it disabled? I found it to be beneficial on mobile for my clients as it relies on the device/browser and not custom styles that might not work on all devices.

**Reply by alexplex** [reply]
I actually tried do edit the flatpickr.js file directly, just to se how it would look, but as far as I could see it didn't do anything. So I figured that there had to be something else I missed.

The reason I wan't to disable the native HTML5 date picker is because it doesn't support place holders. It just displays an ugly YYYY-MM-DD place holder which doesn't look very pleasant. My date pickers need to look presentable in mobile view because they are very prominent throughout my website due to the nature of my business. 

And also, in the future, I would like to be able to edit the date pickers so that users aren't able to pick past dates, for example. The date pickers are used for a booking system with a date picker for arrival and one for departure. But that's a problem for another time. For now, I just want to be able to display my own place holders. Which of course works fine in desktop view but not in mobile view.
